### PR TITLE
Fix multiline strings in DT binding descriptions

### PR DESCRIPTION
**Pull request by mbolivar-nordic**
This PR cleans up various description strings throughout the devicetree bindings.

In general, if you need a multi-line string in a devicetree binding description, you should use this syntax:

```
description: |
  My very long string
  goes here.
  Look at all these lines!
```

This will be rendered properly into the bindings index as a multi-line, fixed-width string, which usually won't require horizontal scrolling unless your lines are too long.

If you have a very short string, just do this:

```
description: my short string
```

That renders fine, too.

However, don't under any circumstances do this:

```
description: Some string
  Multiple additional lines follow
  The whole thing is very long and detailed
```

Don't do this either:

```
description: >
  Some string
  Multiple additional lines follow
  The whole thing is very long and detailed
```

YAML will swallow all of the newlines in these two cases. That makes your description one very long line. This will require tons of horizontal scrolling for people reading the HTML docs. It is not easy to read the results.

### CODE REVIEW

**Comment by mbolivar-nordic** [comment]
More details: http://yaml-multiline.info/

**Review by galak — APPROVED** [review]

**Review by erwango — APPROVED** [review]

**Review by avisconti — APPROVED** [review]
I feel like you took the inspiration after correcting my PR... :D

File: dts/bindings/net/wireless/nordic,nrf21540-fem.yaml [hunk 1]
```diff
@@ -13,63 +13,67 @@ properties:
         type: phandle-array
         required: false
         description: >
```
Comment (henrikbrixandersen) [inline dts/bindings/net/wireless/nordic,nrf21540-fem.yaml:15]:
```suggestion
        description: |
```

File: dts/bindings/net/wireless/nordic,nrf21540-fem.yaml [hunk 2]
```diff
@@ -13,63 +13,67 @@ properties:
...
-            Gpio of the SOC controlling TX_EN pin of the nRF21540
+            GPIO of the SOC controlling TX_EN pin of the nRF21540
     rx-en-gpios:
         type: phandle-array
         required: false
         description: >
```
Comment (henrikbrixandersen) [inline dts/bindings/net/wireless/nordic,nrf21540-fem.yaml:20]:
```suggestion
        description: |
```

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Comment by mbolivar-nordic** [comment]
@henrikbrixandersen addressed, thanks

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**